python/setup.py.in

```diff
@@ -24,7 +24,7 @@ if '${CMAKE_SYSTEM_PROCESSOR}' not in ['arm', 'armv7-a', 'aarch64']:
     setup_requires+=["opencv-python"]
 
 # the prefix is sys.prefix which should always be usr
-paddle_bin_dir = 'local/opt/paddle/bin'
+paddle_bin_dir = 'opt/paddle/bin'
```

Comment:
This should ends up with difference path when use ` cmake install`.
 ` cmake install` :  `/usr/local/opt/paddle/bin`
`pip install` :   `/usr/opt/paddle/bin`